packages/next/client/image.tsx

```diff
@@ -697,54 +756,132 @@ export default function Image({
+    },
+    srcString: srcString,
+    ...rest,
+  }
   return (
-    <span style={wrapperStyle}>
-      {hasSizer ? (
-        <span style={sizerStyle}>
-          {sizerSvgUrl ? (
-            <img
-              style={{
-                display: 'block',
-                maxWidth: '100%',
-                width: 'initial',
-                height: 'initial',
-                background: 'none',
-                opacity: 1,
-                border: 0,
-                margin: 0,
-                padding: 0,
-              }}
-              alt=""
-              aria-hidden={true}
-              src={sizerSvgUrl}
-            />
+    <>
+      {layout === 'raw' ? (
+        <ImageElement {...imgElementArgs} />
+      ) : (
+        <span style={wrapperStyle}>
+          {hasSizer ? (
+            <span style={sizerStyle}>
+              {sizerSvgUrl ? (
+                <img
+                  style={{
+                    display: 'block',
+                    maxWidth: '100%',
+                    width: 'initial',
+                    height: 'initial',
+                    background: 'none',
+                    opacity: 1,
+                    border: 0,
+                    margin: 0,
+                    padding: 0,
+                  }}
+                  alt=""
+                  aria-hidden={true}
+                  src={sizerSvgUrl}
+                />
+              ) : null}
+            </span>
           ) : null}
+          <ImageElement {...imgElementArgs} />
         </span>
+      )}
+      {priority ? (
+        // Note how we omit the `href` attribute, as it would only be relevant
+        // for browsers that do not support `imagesrcset`, and in those cases
+        // it would likely cause the incorrect image to be preloaded.
+        //
+        // https://html.spec.whatwg.org/multipage/semantics.html#attr-link-imagesrcset
+        <Head>
+          <link
+            key={
+              '__nimg-' +
+              imgAttributes.src +
+              imgAttributes.srcSet +
+              imgAttributes.sizes
+            }
+            rel="preload"
+            as="image"
+            href={imgAttributes.srcSet ? undefined : imgAttributes.src}
+            {...linkProps}
+          />
+        </Head>
       ) : null}
+    </>
+  )
+}
+
+const ImageElement = ({
+  imgAttributes,
+  heightInt,
+  widthInt,
+  qualityInt,
+  layout,
+  className,
+  imgStyle,
+  blurStyle,
+  lazy,
+  imgRef,
+  loading,
+  sizes,
+  srcString,
+  config,
+  unoptimized,
+  loader,
+  ...rest
+}: ImageElementProps) => {
+  return (
+    <>
       <img
         {...rest}
         {...imgAttributes}
+        {...(layout === 'raw' && !sizes
+          ? { height: heightInt, width: widthInt }
+          : {})}
         decoding="async"
         data-nimg={layout}
         className={className}
         ref={imgRef}
         style={{ ...imgStyle, ...blurStyle }}
       />
-      {(isLazy || placeholder === 'blur') && (
+      {lazy && 
```

Comment:
```suggestion
      {(lazy || placeholder === 'blur') && 
```

This is still necessary